src/convolution.c

```diff
@@ -212,10 +212,30 @@ enum qnnp_status qnnp_create_convolution2d_nhwc_q8(
       goto error;
     }
 
-    pack_q8dw_w(
-      kernel_height, kernel_width,
-      groups, cr,
-      kernel, bias, convolution->packed_kernel);
+    if (kernel_size == 9) {
+      pack_q8dw_w(
+        kernel_height, kernel_width,
+        groups, cr,
+        kernel, bias, convolution->packed_kernel);
+    } else {
+      // kernel size == 25
+      /* change this later */
```

Comment:
This is suboptimal. I should have a better packing function for it. But I want to do it in coming PR. You can leave your comments here